### DISCUSSION TITLE
Dynamically adjust search results

### DISCUSSION THREAD

**Discussion by Jengro777** [meilisearch/product#533] [Feedback & Feature Proposal]
Hopefully Meilisearch can dynamically adjust search results through Rules like Algolia

Refer to： https://www.algolia.com/doc/guides/managing-results/rules/rules-overview/

**Comment by gmourier** [comment]
Hi @Avey777 👋

Can you tell me more about why you need this?

Thanks!

**Comment by gmourier** [comment]
Hey 👋

I'm locking this discussion in favor of https://github.com/meilisearch/product/discussions/546
